server/tests/src/test/resources/database/sybase.properties

```diff
@@ -0,0 +1,17 @@
+id.column.type=VARCHAR(255)
+data.column.type=IMAGE
+timestamp.column.type=BIGINT
+segment.column.type=BIGINT
+database.mode=CONTAINER
+database.container.name=datagrip/sybase
```

Comment:
You know what I'm going to say here :slightly_smiling_face: 

"`quay.io` if possible"

Reply:
Sure, but at this case there isn't any sybase image on `quay.io`

Reply:
That's fine thinking about it as these DBs are not tested on the CI. If we want to test these via the CI we will need a non docker hub image.